docker/Dockerfile-php56

```diff
@@ -1,23 +1,55 @@
-FROM jolicode/php56:latest
+FROM php:5.6-alpine
 
 ENV SRC_DIR /usr/src/gaufrette
-WORKDIR ${SRC_DIR}/
 
-RUN sudo apt-get update && \
-    sudo apt-get install -y libssh2-1-dev && \
-    sudo rm -rf /var/lib/apt/lists/* && \
-    pecl install mongodb ssh2
+RUN apk add --no-cache --virtual .persistent-deps \
+        git \
+        zlib
 
-COPY composer.json ${SRC_DIR}/
-RUN sudo chown -R travis ${SRC_DIR} && \
-    composer global require hirak/prestissimo && \
-    composer update --prefer-lowest --prefer-stable && \
-    composer require microsoft/windowsazure:~0.4.3 && \
-    rm -rf ~/.composer/cache/*
+# PHP extensions
+ENV APCU_VERSION="4.0.11" \
+    MONGODB_VERSION="1.2.9" \
+    SSH2_VERSION="0.13"
 
-VOLUME ${SRC_DIR}/vendor
+RUN set -xe \
+    && apk add --no-cache --virtual .build-deps \
+        $PHPIZE_DEPS \
+        libssh2-dev \
+        zlib-dev \
+    && docker-php-ext-install \
+        zip \
+    && pecl install \
+        apcu-${APCU_VERSION} \
+        mongodb-${MONGODB_VERSION} \
```

Comment:
Why do you specify versions here? We should always install the most up-to-date version.

Reply:
Just in case they release a new version that is broken/incompatible with the current PHP version.

Reply:
IMHO, that's a best practice for projects but not for library. It's the same problem as we had some years ago with `composer.lock` and php libs. I think we shouldn't force extension versions unless we've good reason to do so, like `ssh2`, which is broken.